data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1640,7 +1642,9 @@ wl.Descriptions():new_tribe {
             -- TRANSLATORS: Lore helptext for an Atlantean production site:  Mill
             lore = pgettext("atlanteans_building", "We invented this vertical axis design first, the sawmill just copied our ideas but never acknowledged our claim."),
             -- TRANSLATORS: Lore author helptext for an Atlantean production site: Mill
-            lore_author = pgettext("atlanteans_building", "Miller arguing with a sawyer")
+            lore_author = pgettext("atlanteans_building", "Miller arguing with a sawyer"),
+            -- TRANSLATORS: Performance helptext for an Atlantean production site: Mill
+            performance = pgettext("atlanteans_building", "If this building is fully supplied and all kinds of flour are needed by the economy, production of one cornmeal takes %1$s and one blackroot flour takes %2$s on average. If only one kind of flour is needed by the economy, production of one cornmeal takes %3$s and one blackroot flour takes %4$s on average."):bformat(format_seconds(44), format_seconds(44), format_seconds(22), format_seconds(22))
```

Comment:
```suggestion
            performance = pgettext("atlanteans_building", "If this building is fully supplied and both kinds of flour are needed by the economy, production of one sack of each kind of flour takes %1$s on average. If only one kind of flour is needed by the economy, its production takes %2$s on average."):bformat(format_seconds(44), format_seconds(22))
```

Reply:
But doesn't this one have the issue that IIRC it only produces blackroot flour when cornmeal is not needed? (or was it the other way around?)